ksqldb-functional-tests/src/main/java/io/confluent/ksql/test/tools/TestCaseBuilder.java

```diff
@@ -75,7 +75,7 @@ private static TestCase createTest(
       );
 
       final Optional<Matcher<Throwable>> ee = test.expectedException()
-          .map(een -> een.build(Iterables.getLast(statements)));
+          .map(ExpectedExceptionNode::build);
```

Comment:
Backported from 6.1 that sets up tests differently (also changes in `ExpectedExceptionNode`)